Extensions/notifications_plus.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Notifications+ **//
-//* VERSION 1.5 REV C **//
+//* VERSION 1.5.1 **//
```

Comment:
Version number was incremented incorrectly.  `REV C` + 1 would be `REV D`, which maps to `1.5.4`.